river/InputManager.zig

```diff
@@ -151,3 +151,14 @@ fn handleNewInput(listener: ?*c.wl_listener, data: ?*c_void) callconv(.C) void {
     // TODO: suport multiple seats
     self.defaultSeat().addDevice(device);
 }
+
+/// Returns a seat matching a wlr_seat.
+pub fn seatFromWlrSeat(self: *Self, wlr_seat: *c.wlr_seat) ?*Seat {
```

Comment:
We store a pointer to the river `Seat` in the `data` field of the `wlr_seat`, this function shouldn't be needed.